contributing/release-schedule.md

```diff
@@ -9,6 +10,7 @@
 - [Release Cadence](#release-cadence)
   - [Detailed schedule](#detailed-schedule)
   - [Release Freezes](#release-freezes)
+[/filter]
```

Comment:
> do you know why this should be indented?

Prettier thinks the line with `[/filter]` belongs to the list and not to the outer `[filter format="none"]`. You can fix this by adding blank lines.

```markdown
[filter format="none"]

- [Release Channels](#release-channels)
  - [Nightly](#nightly)
  - [Weekly](#weekly)
    - [Experimental and Beta channels](#experimental-and-beta-channels)
  - [Long-Term Stable (lts)](#long-term-stable-lts)
- [Determining if your change is in a release](#determining-if-your-change-is-in-a-release)
- [Release Cadence](#release-cadence)
  - [Detailed schedule](#detailed-schedule)
  - [Release Freezes](#release-freezes)

[/filter]
```